src/api/app/models/cloud/params/ec2.rb

```diff
@@ -15,6 +17,14 @@ def self.build(params)
         new(params.slice(:region, :ami_name))
       end
 
+      def to_xml(options = {})
```

Comment:
Why is this necessary? I'm missing where it is used?

Reply:
This is used in the [to_xml method of the UploadJob model](https://github.com/openSUSE/open-build-service/pull/4691/files#diff-53550eae0633ce2e45f0e84869c7fb86R50)